### PR TITLE
Add SharePoint Server Links

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/prepare-tls-1.2-in-office-365.md [hunk 1]
```diff
@@ -75,3 +75,6 @@ The following resources provide guidance to help make sure that your clients are
 - [Exchange Server TLS guidance Part 2: Enabling TLS 1.2 and Identifying Clients Not Using It](https://techcommunity.microsoft.com/t5/exchange-team-blog/exchange-server-tls-guidance-part-2-enabling-tls-1-2-and/ba-p/607761)
 - [Exchange Server TLS guidance Part 3: Turning Off TLS 1.0/1.1](https://techcommunity.microsoft.com/t5/exchange-team-blog/exchange-server-tls-guidance-part-3-turning-off-tls-1-0-1-1/ba-p/607898)
 - [Enable TLS 1.1 and TLS 1.2 support in Office Online Server](/officeonlineserver/enable-tls-1-1-and-tls-1-2-support-in-office-online-server)
+- [Enable TLS and SSL support in SharePoint 2013](https://docs.microsoft.com/en-us/sharepoint/security-for-sharepoint-server/enable-tls-and-ssl-support-in-sharepoint-2013)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prepare-tls-1.2-in-office-365.md]:
```suggestion
- [Enable TLS and SSL support in SharePoint 2013](/sharepoint/security-for-sharepoint-server/enable-tls-and-ssl-support-in-sharepoint-2013)
```

File: microsoft-365/compliance/prepare-tls-1.2-in-office-365.md [hunk 2]
```diff
@@ -75,3 +75,6 @@ The following resources provide guidance to help make sure that your clients are
...
+- [Enable TLS 1.1 and TLS 1.2 support in SharePoint Server 2016](https://docs.microsoft.com/en-us/sharepoint/security-for-sharepoint-server/enable-tls-1-1-and-tls-1-2-support-in-sharepoint-server-2016)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prepare-tls-1.2-in-office-365.md]:
```suggestion
- [Enable TLS 1.1 and TLS 1.2 support in SharePoint Server 2016](/sharepoint/security-for-sharepoint-server/enable-tls-1-1-and-tls-1-2-support-in-sharepoint-server-2016)
```

File: microsoft-365/compliance/prepare-tls-1.2-in-office-365.md [hunk 3]
```diff
@@ -75,3 +75,6 @@ The following resources provide guidance to help make sure that your clients are
...
+- [Enable TLS 1.1 and TLS 1.2 support in SharePoint Server 2019](https://docs.microsoft.com/en-us/sharepoint/security-for-sharepoint-server/enable-tls-1-1-and-tls-1-2-support-in-sharepoint-server-2019)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prepare-tls-1.2-in-office-365.md]:
```suggestion
- [Enable TLS 1.1 and TLS 1.2 support in SharePoint Server 2019](/sharepoint/security-for-sharepoint-server/enable-tls-1-1-and-tls-1-2-support-in-sharepoint-server-2019)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly change the links to relative links as indicated. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder

**Pull request merged**